### DISCUSSION TITLE
app.css/app.js Failed to load resource net::ERR_ABORTED 500

### DISCUSSION THREAD

**Discussion by theshubhamchugh** [filamentphp/filament#6021] [General]
I am facing an issue with my production server for my website, https://companiesnetworth.com/admin/login. When I try to access the website, I am getting an error message that says "app.css/app.js Failed to load resource". However, the website is working fine on my local machine with Laravel Sail.

I have tried to resolve the issue by myself, but I couldn't find a solution. I even tried changing the Nginx configuration, but the issue still persists. After some research, I found that this issue could be caused by a problem with the production server configuration. However, I am not sure what exactly is causing the problem.

Solution i tried: 
URL::forceScheme('https') Added

Added in nginx file
location ^~ /filament {
    try_files $uri $uri/ /index.php?$query_string;
}

https://github.com/livewire/livewire/issues/242
https://github.com/filamentphp/filament/issues/5276

![Screenshot 2023-03-24 at 6 49 49 PM](https://user-images.githubusercontent.com/5415880/227532017-dcf3cf1e-62ca-4039-95d5-b57dc508168b.png)

**Comment by danharrin** [comment]
What is the 500 error

**Comment by theshubhamchugh** [comment]
I attempted to install a fresh Laravel instance with only the Filament package on my test server, and I have also tried various methods such as making changes to Nginx. However, despite these efforts, I am still experiencing issues. Currently, I am utilizing [runcloud.io](http://runcloud.io) to manage my VPS.

Test domain URL: http://test.unep8rqsox-wg96g9ek53oy.p.temp-site.link/admin


![Screenshot 2023-03-25 at 1 12 51 PM](https://user-images.githubusercontent.com/5415880/227704696-178e8df4-3a82-4963-ae31-493c96fb80fb.png)



Server Log: 

2023/03/25 07:47:18 [error] 25572#25572: *49 upstream prematurely closed FastCGI request while reading upstream, client: 110.235.219.71, server: test.unep8rqsox-wg96g9ek53oy.p.temp-site.link, request: "GET /filament/assets/app.js?id=8fc482ef70c5397afa26a4ce85f38519 HTTP/1.1", upstream: "fastcgi://unix:/var/run/test.sock:", host: "test.unep8rqsox-wg96g9ek53oy.p.temp-site.link", referrer: "http://test.unep8rqsox-wg96g9ek53oy.p.temp-site.link/admin/login"

**Comment by theshubhamchugh** [comment]
After a month of conversation, the Runcloud team provided a solution.
![Screenshot 2023-05-04 at 6 40 56 PM](https://user-images.githubusercontent.com/5415880/236216766-2d578e88-17de-4071-b771-da14e507190c.png)

Please try to mention about ignore_user_abort function in the documentation.

**Reply by danharrin** [reply]
Please submit a trick to our website, I do not want to add platform-specific recommendations to our docs when barely anyone uses shared hosting
